src/Core/src/ImageSources/IImageSourceService.cs

```diff
@@ -11,6 +11,11 @@ public interface IImageSourceService
 			IImageSource imageSource,
 			Android.Content.Context context,
 			CancellationToken cancellationToken = default);
+
+		Task<IImageSourceServiceResult<Android.Graphics.Drawables.Drawable>?> LoadDrawableAsync(
```

Comment:
This doesn't need to return a drawable. This is a different api with the image view.

Reply:
Ah, I see it is on the result api... Should we support a non-result one? It can be a base.